lib/routes/account.js

```diff
@@ -536,6 +536,11 @@ module.exports = function (
           response.keyFetchToken = keyFetchToken.data.toString('hex')
 
           if (doSigninConfirmation) {
+            log.activityEvent('account.signin.verify.start', request, {
```

Comment:
To ensure we get accurate metrics, I think we should only log this when `shouldSendVerifyLoginEmail` is true, which IIUC is not precisely the same as `doSigninConfirmation` because it also checks whether keys are being requested.